README.md

```diff
@@ -3,202 +3,212 @@
 [ci-badge]: https://travis-ci.org/firebase/firebase-jobdispatcher-android.svg?branch=master
 [ci-link]: https://travis-ci.org/firebase/firebase-jobdispatcher-android
 
-The Firebase Android JobDispatcher is a library that provides a high-level wrapper around job
-scheduling engines on Android, starting with the [GCM Network Manager][nts].
+The Firebase JobDispatcher is a library for working with the [JobScheduler][] compatibility library
```

Comment:
s/ for working with / to schedule future and periodic background jobs in your app. It uses